05_testing_and_ci/python_testing_slides.md

```diff
@@ -0,0 +1,74 @@
+---
+type: slide
+slideOptions:
+  transition: slide
+  width: 1400
+  height: 900
+  margin: 0.1
+---
+
+<style>
+  .reveal strong {
+    font-weight: bold;
+    color: orange;
+  }
+  .reveal p {
+    text-align: left;
+  }
+  .reveal section h1 {
+    color: orange;
+  }
+  .reveal section h2 {
+    color: orange;
+  }
+</style>
+
+# Testing a Python Code
```

Comment:
```suggestion
# Testing a Python code
```
Later on it is sentence case for headlines. Ignore this comment if this is done on purpose.